.travis/environments/travis/inventory.ini

```diff
@@ -25,6 +25,9 @@ localhost ansible_connection=local
 [elasticsearch]
 localhost ansible_connection=local elasticsearch_node_name=es0
 
+[es_coordinator]
+localhost ansible_connection=local elasticsearch_node_name=es0
+
```

Comment:
I might suggest removing this so that tests continue to reflect that this group is optional.

Reply:
Sure Danny. thanks.